proposals/0293-extend-property-wrappers-to-function-and-closure-parameters.md

```diff
@@ -500,20 +511,89 @@ Exposing the property-wrapper storage type through the type system has the follo
 * It prohibits the use of initializer arguments in the wrapper attribute. There's no point in declaring a wrapper as `@Asserted(.greaterOrEqual(1))` if any client can simply pass an `Asserted` instance with a completely different validation.
 * It removes API control from both the property wrapper author and the author of the wrapped-argument function.
 
-Keeping the property-wrapper storage type private is consistent with how property wrappers work today. Unless a property wrapper projects its storage type via `projectedValue`, the storage type itself is meant to be private, implementation detail that cannot be accessed by API clients.
+Keeping the property-wrapper storage type private is consistent with how property wrappers work today. Unless a property wrapper projects its storage type via `projectedValue`, the storage type itself is meant to be a private implementation detail inaccessible to API clients.
 
 ## Future directions
 
+### The impact of formalizing separate property wrapper models
```

Comment:
I don't think this is really a future direction. In my opinion, it would fit in better with a "don't formalize the API-level vs implementation-detail distinction" alternatives-considered section (where the fact that it doesn't block future additions is explored). What do you think?

Reply:
This section talks about the impact of this proposal on the future design space for property wrappers. People were really concerned about this on the Pitch 4 thread, so I wrote up a section about it. I agree that it's not talking about a specific feature that could be pursued in the future, but that's not the only purpose of the Future Directions section